include/exadg/poisson/user_interface/enum_types.h

```diff
@@ -68,6 +68,7 @@ enum class Preconditioner
   PointJacobi,
   BlockJacobi,
   AMG,
+  AdditiveSchwarz,
```

Comment:
just a minor comment: I think I would change the order of AMG and AdditiveSchwarz.